types/index.d.ts

```diff
@@ -0,0 +1,8 @@
+// https://github.com/DefinitelyTyped/DefinitelyTyped/blob/b23a36e669fa127d1035e22ca93faab85b98e49f/types/parse/index.d.ts#L11
+// TODO(dplewis): Possibly add @types/node module reference types
+
+/// <reference path="node.d.ts" />
+/// <reference path="react-native.d.ts" />
```

Comment:
Could you remove your name? We don't assign tasks to specific contributors via code comments.
Can we remove the rest of the comments, are they auto-generated?

Reply:
Done. Documentation for reference types directives are [here](https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html#-reference-types-)

Reply:
Reference types are for `parse/node` and `parse/react-native` types. Since they aren’t in the src folder and at the root like index.js they have to be manually created instead of generated.